packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -697,6 +774,9 @@ async function createRedwoodApp() {
     await generateTypes(newAppDir)
   }
 
+  // DELETE THIS 👇
+  tui.drawText(`${RedwoodStyling.info('ℹ')} generated types for ${newAppDir}`)
+
```

Comment:
Did you mean to keep this around? Or should it be removed?

Reply:
😩 I thought I had grabbed all of these ... thanks for catching